Documentation/compatibility/workflow-designer-accessibility.md

```diff
@@ -0,0 +1,123 @@
+## Accessibility improvements in WF workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
```

Comment:
Initialize correlation window for --> The initialize correlation window, for
InitializeCorrelation --> `T:System.ServiceModel.Activities.InitializeCorrelation` 

Also, is this a fairly complete list, or just a few "for example" items. If the latter, can it be expanded to be more comprehensive to have approximately the same level of coverage as [WinForms accessibility improvements](https://github.com/Microsoft/dotnet/blob/master/Documentation/compatibility/winforms-accessibility-changes-471.md)?